examples/05-Legacy/00-overview_example.py

```diff
@@ -1,15 +1,15 @@
 """
 .. _ref_overview_example:
 
-PyDPF-Post overview
-===================
-This example provides an overview of how you use PyDPF-Post.
+Use basic features
+==================
+This example shows how to use basic features of the legacy PyDPF-Post API.
```

Comment:
The examples in the Legacy folder don't necessarily seem logically ordered. Consider visiting this section of examples and reordering them. For example, 01-introduction.py and 09-basics.py seem like they should appear earlier in this section's list of examples.

Reply:
@PipKat @Revathyvenugopal162 I think there is a way to tell sphinx-gallery to order examples a certain way. We need to move away from using numbers in folder names as this is hardly maintainable.

Reply:
@PProfizi I think Jorge recommended we did this a while back for search engine optimization reasons. I'll be curious to see what @Revathyvenugopal162 knows about other/better ways of doing this.

Reply:
@PipKat in that case we can do a reordering in another PR dedicated to changing the sorting option and deprecating the need for numbered folders and files, both for legacy and non-legacy.